src/main/java/de/hysky/skyblocker/skyblock/itemlist/recipebook/SkyblockRecipeResults.java

```diff
@@ -143,6 +152,7 @@ private void drawRecipeDisplay(DrawContext context, TextRenderer textRenderer, i
 
 		//Draw the arrow that points to the recipe's result
 		context.drawTextWithShadow(textRenderer, "▶", x + 96, y + 90, 0xaaffffff);
+		if (recipeIcon != null) context.drawItem(recipeIcon, x + 115, y + 61);
```

Comment:
Forging time is not displayed in the recipe book. This could probably be added with a call to `SkyblockRecipe#render` here.